algorithms/hhl/hhl/hhl.ipynb

```diff
@@ -1,982 +1,1292 @@
 {
```

Comment:
<p>\lambda_j are eigenvalues of a unitary U ---&gt;</p><p>\lambda_j are eigenvalues of a matrix A</p><p>or</p><p>\lambda_j are eigenphases of a unitary U</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Corrected tilde sign in Promise.</p>